### PR TITLE
Plasmaman command sprites

### PR DESCRIPTION
**Pull request by EightBitrous**
all the sprites and updated code for them, still need to make the roles actually start with said suits.

## About The Pull Request
Adds the following:
CE plasmaman suit and Helmet
CMO plasmaman suit and Helmet
RD plasmaman suit and Helmet

## Why It's Good For The Game
This update will help players be able to differentiate between command plasmemes and assistant plasmemes, considering they use the same sprite at the moment. The current sprite is also eye cancer.
command-Plasmaman

## Changelog
:cl:
add: new plasmaman suits for command
/:cl:

### CODE REVIEW

**Comment by GuillaumePrata** [comment]
I kinda want to play Plasmeme RD now just for that suit, good work.

**Comment by ghost** [comment]
you wanna pull up code/modules/mob/living/carbon/human/species_types/plasmaman.dm and scroll down a little - you can see where it assigns roundstart outfits for plasmamen there

**Comment by ghost** [comment]
AFAIK this one contains broken objects - waiting on a fix

**Comment by EightBitrous** [comment]
Problems have been fixed, ready to go

File: code/modules/clothing/under/jobs/Plasmaman/engineering.dm [hunk 1]
```diff
@@ -11,3 +11,8 @@
 	icon_state = "atmos_envirosuit"
 	item_state = "atmos_envirosuit"
 
+/obj/item/clothing/under/plasmaman/ce
+	name = "CE plasma envirosuit"
+	desc = "An air-tight suit designed to be used by plasmamen exployed as the chief engineer, the usual purple stripes being replaced by ce's infinitly better yellow."
```
Comment (Saliferous) [inline code/modules/clothing/under/jobs/Plasmaman/engineering.dm]: exployed -> employed
infinitly -> infinitely 
Just a minor typo, no actual issue

**Review by Saliferous — APPROVED** [review]
Checks out, just minor typos in the desc of suits.

File: code/modules/clothing/under/jobs/Plasmaman/engineering.dm (hunk 1 above)
Comment (EightBitrous) [reply code/modules/clothing/under/jobs/Plasmaman/engineering.dm]: I'll go ahead an fix those real quick

File: code/modules/clothing/spacesuits/plasmamen.dm [hunk 2]
```diff
@@ -146,6 +146,24 @@
 		var/datum/action/A=X
 		A.UpdateButtonIcon()
 
+/obj/item/clothing/head/helmet/space/plasmaman/cmo
+	name = "CMO plasma envirosuit helmet"
+	desc = "A plasmaman containment helmet designed for the chief medical officer."
```
Comment (ghost) [inline code/modules/clothing/spacesuits/plasmamen.dm]: Inconsistent capitalisation with 'chief medical officer' then 'Research Director'. Can you capitalise all the job titles pls?

File: code/modules/clothing/under/jobs/Plasmaman/medsci.dm [hunk 3]
```diff
@@ -33,3 +33,15 @@
 	desc = "A plasmaman envirosuit designed for chemists."
 	icon_state = "chemist_envirosuit"
 	item_state = "chemist_envirosuit"
+
+/obj/item/clothing/under/plasmaman/rd
+	name = "RD plasma envirosuit"
+	desc = "A suit worn by the Researh Director. preferably when they can only breathe plasma."
```
Comment (ghost) [inline code/modules/clothing/under/jobs/Plasmaman/medsci.dm]: Researh Director - typo.

**Comment by ghost** [comment]
Obvs I cant see or test the DMI files, so have you tested that they all spawn and display correctly?

If so @TheSwain aside from some small typos, this seems legit.

File: code/modules/clothing/spacesuits/plasmamen.dm (hunk 2 above)
Comment (EightBitrous) [reply code/modules/clothing/spacesuits/plasmamen.dm]: Sure, I will go ahead and do that now

**Comment by EightBitrous** [comment]
ce1
ce2
ce3
ce4
cmo1
cmo2
cmo3
cmo4
rd1
rd2
rd3
rd4


All sprites tested in-game

**Comment by ghost** [comment]
Looking sharp man, real pretty.

Fix up those typos and I'd say this is good to merge.

**Comment by EightBitrous** [comment]
The spelling errors have been fixed, good to go.

**Comment by ghost** [comment]
Seems like they have. Looking forward to seeing these ingame.

@TheSwain This all looks good.

**Comment by ghost** [comment]
Cosmic pointed out that this is using the base .dmi files and that's NO GOOD.

I'll run you through how to change that tommorow.

**Pull request closed without merging**